spotui/src/Logging.py

```diff
@@ -2,5 +2,18 @@
 import os
 
 user_config_dir = os.path.expanduser("~")
-logging.basicConfig(filename=user_config_dir + '/.cache/spotui.log', filemode='w',
+filename = user_config_dir + "/.cache/spotui.log"
+if not os.path.exists(os.path.dirname(filename)):
+  try:
+    os.makedirs(os.path.dirname(filename))
+  except OSError as exc:  # prevent race condition
+    if exc.errno != errno.EEXIST:
+      raise
+      
+# if log file doesn't exist
+if not os.path.isfile(filename):
+  with open('sample.csv', 'w') as creating_new_log_file: 
+    pass
+        
+logging.basicConfig(filename, filemode='w',
```

Comment:
`logging.basicConfig` doesn't take any position args. If you look at the original code (or just check the function signature) you'll see it expects filename as a keyword arg. I'd suggest running your code at least once just to sanity check this kind of thing